bandit/core/manager.py

```diff
@@ -264,9 +264,11 @@ def _parse_file(self, fname, fdata, new_files_list):
             lines = data.splitlines()
             self.metrics.begin(fname)
             self.metrics.count_locs(lines)
-            if self.ignore_nosec:
-                nosec_lines = set()
-            else:
+
+            nosec_lines = set()
```

Comment:
The code here has changed enough after you rebase, that your changes might no longer have an effect. Please rebase and re-access whether this is still an improvement.